apps/explorer/src/api/assets/assetsApi.ts

```diff
@@ -106,11 +101,16 @@ export const assetsQueries = createQueriesCollection({
     unverifiedFungibleToken: (assetId: string) => ({
       queryKey: ['unverifiedFungibleToken', assetId],
       queryFn: (): Promise<UnverifiedFungibleTokenMetadata> =>
-        fungibleTokensMetadata.fetch(assetId).then((r) => ({
-          ...r,
-          type: 'fungible',
-          verified: false
-        })),
+        fungibleTokensMetadata.fetch(assetId).then((r) => {
+          const parsedDecimals: number = parseInt(r.decimals)
```

Comment:
No need to add `: number`, it was just an example in the PR review to show that this will be a number :) But it doesn't hurt!